Resources/Prototypes/Entities/Mobs/base.yml

```
@@ -69,21 +69,22 @@
-      behaviors:
-      - !type:SpawnEntitiesBehavior
-        spawnInContainer: true
-        spawn:
-          Ash:
-            min: 1
-            max: 1
-      - !type:BurnBodyBehavior { }
-      - !type:PlaySoundBehavior
-        sound:
-          collection: MeatLaserImpact
+    # GoobStation: Disabled ashing until husking is added
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)